### ISSUE TITLE
Can't see how to get full error message

### ISSUE THREAD

**Issue by fredemmott** [PranayAgarwal/vscode-hack#17]
vscode is only showing me `[Hack] Wrong type hint (17,42)`

Nuclide and hh_client show me:

```
src/fbmarkdown/unparsed-blocks/Context.php:17:42,56: Wrong type hint (Typing[4110])
  src/fbmarkdown/unparsed-blocks/Context.php:17:26,38: This is an object of type Facebook\Markdown\UnparsedBlocks\BlockProducer
  src/fbmarkdown/unparsed-blocks/Context.php:27:5,15: It is incompatible with an object of type Facebook\Markdown\UnparsedBlocks\ListOfItems
```

**Comment by PranayAgarwal** [comment]
Are you using the extension in LSP mode? If so, you'll need to create this issue against the hh_client source.

**Comment by fredemmott** [comment]
This is exposed over LSP, but it might be an extension; nuclide pulls it up in LSP mode. That said, the vim8 LSP plugin also doesn’t make this info available

**Comment by fredemmott** [comment]
This appears to be exposed as a non-standard `relatedLocations` field of the diagnostics.

**Comment by fredemmott** [comment]
https://github.com/Microsoft/language-server-protocol/issues/260 is the proposal for this feature

**Comment by fredemmott** [comment]
This functionality is now in the protocol, but `hh_client`'s support is not compatible. FB task T29277163 to fix this.

**Comment by fredemmott** [comment]
I have a hack-side fix for this (FB D8286533)
